eservice/lib/libpdo_enclave/contract_worker.cpp

```diff
@@ -35,11 +35,12 @@ void ContractWorker::InitializeInterpreter(void)
 
     if (current_state_ == INTERPRETER_DONE)
     {
-        if (interpreter_ != NULL) {
-            delete interpreter_;
+        if (interpreter_ == NULL) {
```

Comment:
Nice style opening a block at the end of the line -- it reminds me of someone! ;)